infrastructure/terraform/modules/deployment/variables.tf

```diff
@@ -1,69 +1,73 @@
 variable "region" {
   description = "AWS region shortname"
-  type = string
-  default = "us-east-1"
+  type        = string
+  default     = "us-east-1"
 }
 
 variable "name" {
   description = "Proper name for this environment"
-  type = string
+  type        = string
 }
 
 variable "environment" {
   description = "Functional name for this environment"
-  type = string
+  type        = string
 }
 
 variable "pubpub_hostname" {
   description = "hostname where pubpub will be addressable (DO NOT include https://)"
-  type = string
+  type        = string
 }
 
 variable "route53_zone_id" {
   description = "Zone ID of route53 zone that is already configured as the NS for your subdomain"
-  type = string
+  type        = string
 }
 
 variable "ecr_repository_urls" {
   description = "URLs for ECR repositories created at a global level"
   type = object({
-    core = string
+    core             = string
     intg_evaluations = string
     intg_submissions = string
-    jobs = string
-    nginx = string
-    root = string
+    jobs             = string
+    nginx            = string
+    root             = string
   })
 }
 
 variable "MAILGUN_SMTP_USERNAME" {
   description = "SMTP Username for Mailgun service"
-  type = string
+  type        = string
 }
 
 variable "MAILGUN_SMTP_HOST" {
   description = "SMTP Hostname for Mailgun service"
-  type = string
-  default = "smtp.mailgun.org"
+  type        = string
+  default     = "smtp.mailgun.org"
 }
 
 variable "MAILGUN_SMTP_PORT" {
   description = "SMTP Network Port for Mailgun service"
-  type = string
-  default = "465"
+  type        = string
+  default     = "465"
 }
 
 variable "NEXT_PUBLIC_SUPABASE_URL" {
   description = "URL to Supabase public address for this install"
-  type = string
+  type        = string
 }
 variable "NEXT_PUBLIC_SUPABASE_PUBLIC_KEY" {
   description = "Supabase anon public key"
-  type = string
+  type        = string
+}
+
+variable "HOSTNAME" {
+  description = "Hostname used by standalone Next app"
 }
```

Comment:
Added this variable to override the default HOSTNAME variable provided by ECS so that the Next app can bind to the proper hostname when starting.